app/experimenter/experiments/api_views.py

```diff
@@ -60,3 +62,24 @@ def update(self, request, *args, **kwargs):
         )
 
         return Response()
+
+
+class ExperimentSendIntentToShipEmailView(UpdateAPIView):
+    lookup_field = "slug"
+    queryset = Experiment.objects.filter(status=Experiment.STATUS_REVIEW)
+
+    def update(self, request, *args, **kwargs):
+        experiment = self.get_object()
+
+        if experiment.review_intent_to_ship:
+            return Response(
+                {"error": "email-already-sent"},
+                status=status.HTTP_409_CONFLICT,
+            )
+
+        email.send_intent_to_ship_email(experiment.id)
```

Comment:
We don't want to send the email right in the view here, wrap it in a task in tasks.py and use Notification to alert the user that it was either sent successfully or failed.

Reply:
I also couldn't make up my mind about this. The problem is that email can fail much later than it's been "sent", so even blocking the modal doesn't really buy you anything. I ended up going with YAGNI, especially since this is an internal site and responsiveness isn't crucial, but I could be convinced either way.